site/content/docs/4.3/components/offcanvas.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: docs
+title: Offcanvas
+description: Toggle the visibility sidebars across your project with a few classes and our JavaScript plugin.
+group: components
+toc: true
+---
+
+## How it works
+
+The offcanvas JavaScript plugin is used to show and hide a sidebar. Buttons or anchors are used as triggers that are mapped to specific elements you toggle. Given how CSS handles animations, you cannot use `margin` & `translate` on a `.offcanvas` element. Instead, use the class as an independent wrapping element.
+
+## Example
+
+Click the buttons below to show and hide another element via class changes:
+
+- `.offcanvas` hides content
+- `.offcanvas.show` shows content
+
+You can use a link with the `href` attribute, or a button with the `data-target` attribute. In both cases, the `data-toggle="offcanvas"` is required.
+
+{{< example >}}
+<p>
+  <a class="btn btn-primary" data-toggle="offcanvas" href="#offcanvasExample" role="button" aria-expanded="false" aria-controls="offcanvasExample">
+    Link with href
+  </a>
+  <button class="btn btn-primary" type="button" data-toggle="offcanvas" data-target="#offcanvasExample" aria-expanded="false" aria-controls="offcanvasExample">
+    Button with data-target
+  </button>
+</p>
+<div class="offcanvas offanvas-push-body bg-dark text-white" tabindex="-1" id="offcanvasExample">
+  <div class="offcanvas-header">
+     <div class="h4 col text-center">Offcanvas</div>
```

Comment:
These examples need reindendation @GeoSot. All of them in this page.

It seems a `</div>` is missing somewhere in all of them.